components/brave_wallet_ui/common/async/handlers.ts

```diff
@@ -816,6 +817,12 @@ handler.on(WalletActions.addFilecoinAccount.type, async (_store: Store, payload:
   return result.success
 })
 
+handler.on(WalletActions.addBitcoinAccount.type, async (_store: Store, payload: AddBitcoinAccountPayloadType) => {
+  const { keyringService } = getAPIProxy()
+  const result = await keyringService.addBitcoinAccount(payload.accountName, payload.networkId, payload.keyringId)
+  return result.success
+})
```

Comment:
Please redefine this as a query within `api.slice.ts` since `handlers.ts` is undergoing deprecation